lib/round.js

```diff
@@ -7,8 +9,9 @@ export class RoundData {
    */
   constructor (index) {
     this.index = index
-    /** @type {import('./preprocess.js').Measurement[]} */
+    /** @type {string[]} */
     this.measurementBatches = []
+    /** @type {Measurement[]} */
     this.measurements = []
```

Comment:
The field `measurementBatches` stores a list of CIDs, see here:

https://github.com/filecoin-station/spark-evaluate/blob/670984007ce2b64177281a1f98ebe611274c8132/lib/preprocess.js#L119